### ISSUE TITLE
Weekly scheduled task runs infinitely

### ISSUE THREAD

**Issue by avainio** [M-Files/VAF.Extensions.Community#138]
Background task that is scheduled to run weekly at certain times ends up running in never ending loop. New task is started immediately when previous task is completed even though task should run only once.
Weekly Scheduled Task is configured with MFiles.VAF.Extensions.Frequency
Problem happens at least with Time zone: FLE Daylight Time

**Comment by CraigHawker** [comment]
A potential resolution is in `24.10.0.5-issue-138`.

**Comment by CraigHawker** [comment]
Fix confirmed in preview release; code merged and issue closed.
